src/main/resources/static/restaurantchoice.css

```diff
@@ -1,6 +1,6 @@
 @charset "ISO-8859-1";
 button {
-  visibility: hidden;
+  /*visibility: hidden;*/
```

Comment:
the button was probably hidden before because it wasn't working. It works now so i commented this out.